### PR TITLE
Vis.  Fix DataGroup mappings for nestedGroups, added visible and showNested to DataGroup as described in vis.js documentation

### PR DESCRIPTION
**Pull request by Airex**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://visjs.org/docs/timeline/#groups
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by slavede** [comment]
Can you add an example in test file? I see it in the documentation that's correct, but it would useful to have definition in test just to prove the test is not failing?

**Comment by adripanico** [comment]
Were 7 commits necessary for this?

**Comment by Airex** [comment]
> Were 7 commits necessary for this?

Does it matter?

**Comment by adripanico** [comment]
> > Were 7 commits necessary for this?
> 
> Does it matter?

Yes, it does matter.

**Comment by Airex** [comment]
> > > Were 7 commits necessary for this?
> > 
> > 
> > Does it matter?
> 
> Yes, it does matter.

I forgot to make clean fork, as previous contained my prev attempts to contribute. Sorry for that

**Comment by Airex** [comment]
> Can you add an example in test file? I see it in the documentation that's correct, but it would useful to have definition in test just to prove the test is not failing?

ready

File: types/vis/index.d.ts [hunk 1]
```diff
@@ -107,8 +107,11 @@ export interface DataGroup {
   style?: string;
   subgroupOrder?: string | (() => void);
   title?: string;
-  nestedGroups?: number[];
+
```
Comment (MichaelBitard) [inline types/vis/index.d.ts]:
```suggestion
```

**Review by MichaelBitard — APPROVED** [review]

File: types/vis/index.d.ts (hunk 1 above)
Comment (rbuckton) [reply types/vis/index.d.ts]: @MichaelBitard: Should this be fixed before I merge?

**Comment by rbuckton** [comment]
Thanks for the contribution!

File: types/vis/index.d.ts (hunk 1 above)
Comment (MichaelBitard) [reply types/vis/index.d.ts]: It was a suggestion and I already approved the PR, so it was up to you ;)

**Pull request merged**